communities/src/lib.rs

```diff
@@ -54,11 +54,8 @@ pub mod pallet {
 	use frame_support::pallet_prelude::*;
 	use frame_system::pallet_prelude::*;
 
-	const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Comment:
Same here, I think we need this. @pifragile am I correct?

Reply:
Ahh, is this maybe also just a rebase error?

Reply:
Yeah, I lost track of what's where during multiple merge conflicts, sorry about this! :sweat_smile:

Reply:
Fixed in [4dc5183](https://github.com/encointer/pallets/pull/322/commits/4dc518340aadcdb64148393ff27ac6ee526df993).